wgpu-hal/src/lib.rs

```diff
@@ -1295,11 +1325,50 @@ pub trait CommandEncoder: WasmNotSendSync + fmt::Debug {
 
     // render passes
 
-    // Begins a render pass, clears all active bindings.
+    /// Begin a new render pass, clearing all active bindings.
+    ///
+    /// This clears any bindings established by the following calls:
+    ///
+    /// - [`set_bind_group`]
+    /// - [`set_push_constants`]
+    /// - [`begin_query`]
+    /// - [`set_render_pipeline`]
+    /// - [`set_index_buffer`]
+    /// - [`set_vertex_buffer`]
+    ///
+    /// [`set_bind_group`]: CommandEncoder::set_bind_group
+    /// [`set_push_constants`]: CommandEncoder::set_push_constants
+    /// [`begin_query`]: CommandEncoder::begin_query
+    /// [`set_render_pipeline`]: CommandEncoder::set_render_pipeline
+    /// [`set_index_buffer`]: CommandEncoder::set_index_buffer
+    /// [`set_vertex_buffer`]: CommandEncoder::set_vertex_buffer
```

Comment:
style: This form seems unnecessarily duplicative when compared to providing the link target on the same line (i.e., ``[`ident`](link)``). The current style is consistent with other linking forms, but I think that the consistency would only be valuable if the links were kept in a single region (i.e., for each of updating symbols).

suggestion:

```suggestion
    /// - [`set_bind_group`](CommandEncoder::set_bind_group)
    /// - [`set_push_constants`](CommandEncoder::set_push_constants)
    /// - [`begin_query`](CommandEncoder::begin_query)
    /// - [`set_render_pipeline`](CommandEncoder::set_render_pipeline)
    /// - [`set_index_buffer`](CommandEncoder::set_index_buffer)
    /// - [`set_vertex_buffer`](CommandEncoder::set_vertex_buffer)
```